tests/unit/sagemaker/model/test_model.py

```diff
@@ -315,7 +315,25 @@ def test_create_sagemaker_model_accelerator_type(prepare_container_def, sagemake
     model._create_sagemaker_model(INSTANCE_TYPE, accelerator_type=accelerator_type)
 
     prepare_container_def.assert_called_with(
-        INSTANCE_TYPE, accelerator_type=accelerator_type, serverless_inference_config=None
+        INSTANCE_TYPE,
+        accelerator_type=accelerator_type,
+        serverless_inference_config=None,
+        accept_eula=None,
+    )
+
+
+@patch("sagemaker.model.Model.prepare_container_def")
+def test_create_sagemaker_model_with_eula(prepare_container_def, sagemaker_session):
+    model = Model(MODEL_IMAGE, MODEL_DATA, name=MODEL_NAME, sagemaker_session=sagemaker_session)
+
+    accelerator_type = "ml.eia.medium"
+    model.create(INSTANCE_TYPE, accelerator_type=accelerator_type, accept_eula=True)
+
+    prepare_container_def.assert_called_with(
+        INSTANCE_TYPE,
+        accelerator_type=accelerator_type,
+        serverless_inference_config=None,
+        accept_eula=True,
```

Comment:
add a unit test for `False` case please

Reply:
`question: what happens if accept_eula is False ? Does it use the attribute?`-- Good catch, I will fix this and add the test

`can you create a separate utility for this + unit test it please?` separate utility for what exactly? there already is a unit test that the log does not get emitted for JS models